src/arcaflow_plugin_sdk/schema.py

```diff
@@ -1757,15 +1786,15 @@ class StringEnumSchema(_JSONSchemaGenerator, _OpenAPIGenerator):
     ... })
     >>> s.valid_values()
     ['apple', 'banana', 'orange']
-    """
+    """  # noqa: E501
```

Comment:
Having passed many of these, I finally got around to looking up E501. While most people may run an IDE (or even vi) wider than 80 columns, it's worth mentioning that in a side-by-side GitHub diff display, line 1776 *wraps* on both sides, making the review just ever so slightly more difficult. Keeping within the recommended line width really does have some value ...

Reply:
I couldn't find any information about what folks do for Sphinx docstrings that have to get converted into restructured text. Is it no different than regular docstrings?

Reply:
We haven't tried to auto-generate internal API documentation, and I have no experience with Sphinx; but if it requires specific line wrapping (especially if it requires making lines wider than language good practice) in a docstring, that's an extremely fragile system and I wouldn't want to stand too close to it!

(For that matter, I despise Sphinx and Google parameter style conventions, and the type information is completely redundant with modern Python type hints. Any API documentation system worth even considering would be driven by a free-form docstring description with argument/return/raises documentation dynamically inspected/introspected from the code. 😀 )

Reply:
I think you have a good point